scripts/install.sh

```diff
@@ -766,8 +766,16 @@ function get_user_config() {
         return
     fi
 
-    # extract install_token and strip quotes
-    grep -m 1 install_token "${file}" \
+    # extract installation_token and strip quotes
+    # fallback to deprecated install_token
+    grep -m 1 installation_token "${file}" \
+        | sed 's/.*installation_token:[[:blank:]]*//' \
+        | sed 's/[[:blank:]]*$//' \
+        | sed 's/^"//' \
+        | sed "s/^'//" \
+        | sed 's/"$//' \
+        | sed "s/'\$//" \
+    || grep -m 1 install_token "${file}" \
```

Comment:
why do we need two `greps` here and not just `grep` for `installation token` in the file?

Reply:
Backward compatibility. We want to support previous installations